services/ui-src/src/utils/reports/reports.test.ts

```diff
@@ -112,3 +118,39 @@ describe("Sort Reports", () => {
     expect(sortReportsOldestToNewest(unsortedReports)).toEqual(sortedReports);
   });
 });
+
+describe("Test copyAdminDisabledStatusToForms", () => {
+  const newReportJson: ReportJson = {
```

Comment:
We actually have some mockReportJson available [here in setupJest.tsx](https://github.com/CMSgov/mdct-mcr/blob/ddb762731824513e0bd8a78e977c475709a4d036/services/ui-src/src/utils/testing/setupJest.tsx#L230-L235) you could use.